src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -25,7 +25,8 @@ const initState: ComponentState = {
     isPrebuilt: false,
     isMultivalueVal: false,
     isNegatableVal: false,
-    isEditing: false,
+    isEditing: false,  
+    enumValues: [undefined, undefined, undefined, undefined, undefined],
```

Comment:
Why did we have to use `undefined` instead of `''` ? It seems unnatural given we expect strings to be the only type of input to text field.

I'm assuming you put multiple entries here to show the user that they are expected to enter more than 1?